content/en/docs/reference/instrumentation/slis.md

```diff
@@ -0,0 +1,91 @@
+---
+reviewers:
+- logicalhan
+title: Kubernetes Component SLIs
+content_type: reference
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+The Component SLIs feature allows enabling of a new metrics endpoint for each Kubernetes
+binary. This metric endpoint is exposed on the standard port of each binary at the endpoint
+`metrics/slis`. 
+
+<!-- body -->
+
+## SLI Metrics
+
+We expose two metrics; a gauge (which represents the current state of the healthcheck) and a
+counter (which records the cumulative counts that we've observed for each healthcheck state).
```

Comment:
(nit)

We [avoid using we](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)

Is this right?
```suggestion
With SLI metrics enabled, each Kubernetes component exposes two metrics,
labelled per healthcheck:

- a gauge (which represents the current state of the healthcheck)
- a counter (which records the cumulative counts observed for each healthcheck state)
```